index.html

```diff
@@ -38,7 +38,133 @@ <h2>LET'S CONNECT</h2>
             </section>
         </main>
         </div>
-    </div>
-    
+        
+    </div>  
+</div>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio.png" alt="">
+    <h1>Tonic</h1>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
+
+<section class="hero">
+<div class="card">
+    <img src="img/Snapshoot Portfolio(1).png" alt="">
+    <h1>Multi-Post
+        Stories</h1>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+</section>
```

Comment:
I have found the different font-sizes that are not used in accordance to the Figma Design. Kindly take a look.
1. The size in Figma is `32px`, Yours is `34px`
2. The size in Figma is `13px`, Yours is larger
3. Your alignment of the design is on the left. Kindly align it to the center.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221021699-5819c282-91e2-471a-ad15-70c39bada7ca.png) | ![image](https://user-images.githubusercontent.com/61361037/221022184-2f9ffc7a-492a-40cb-b5a6-c07381740fe8.png)|